shell/assets/translations/en-us.yaml

```diff
@@ -4991,6 +4995,10 @@ promptRollback:
   multipleWorkloadError: "Only one workload can be rolled back at a time."
   singleRevisionBanner: There are no revisions to roll back to.
 
+promptRedeploy:
+  title: Are you sure?
+  attemptingToRedeploy: "You are attempting to redeploy the {type} {names}"
```

Comment:
I think that the body of the modal should provide more context about the consequence of the action being taken, we should gather feedback on how to best word this. 

I'm thinking that something along the lines of "Redeploying {names} will restart your workloads and may cause temporary downtime."

Reply:
Good suggestion, I’ve fine-tuned the wording, updated in commit https://github.com/rancher/dashboard/pull/14482/commits/d022a48566ddf5d0fde1e21a4518951ac34de214. 
Let me know if you have any other phrasing ideas.
![Screenshot 2025-06-11 at 11 13 57 AM (2)](https://github.com/user-attachments/assets/034d47c4-3223-48f2-b5d6-0e9ccadfc970)